holidayextras/ui-toolkit
# flatten structure, export, and lint

#### What does this PR do? (please provide any background)
Flattens the structure out to how we work now, exports js instead of jsx. Everything is linted now too.
#### What tests does this PR have?
All tests updated in this pr. No new tests added, I've only updated the paths in the tests.
#### How can this be tested?
Run the tests, generate the docs with `npm run docs`, include this branch, check I've not done anything daft.
#### Screenshots / Screencast
#### What gif best describes how you feel about this work?
---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
